examples/material_gallery/lib/main.dart

```diff
@@ -6,16 +6,20 @@ import 'package:flutter/material.dart';
 
 import 'chip_demo.dart';
 import 'gallery_page.dart';
+import 'picker_demo.dart';
 import 'widget_demo.dart';
 
 final List<WidgetDemo> _kDemos = <WidgetDemo>[
-  kChipDemo
+  kChipDemo,
+  kPickerDemo,
```

Comment:
I would think we'd have a separate DatePicker and a TimePicker, and corresponding demos.